### ISSUE TITLE
Cubital Tunnel Syndrome 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10255]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 30855847 Cubital tunnel syndrome is neuropathy of the ulnar nerve causing symptoms of numbness and shooting pain along the medial aspect of the forearm, also including the medial half of the fourth digit and the fifth digit. It is caused by compression of the ulnar nerve at the elbow region. 
PMID: 36175329 Compressive neuropathies may ensue as a result of the tissue enlargement. For example, median nerve compression may lead to carpal tunnel syndrome, requiring surgical release. The median nerve is more commonly affected than the ulnar nerve. Cubital tunnel syndrome is a recognized complication of ulnar nerve enlargement in macrodactyly

**Parent term (use hpo.jax.org/app)**
Entrapment neuropathy of the ulnar nerve at elbow HP:0012186

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Macrodactyly of fingers ORPHA:295044

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f I think that [Entrapment neuropathy of the ulnar nerve at elbow HP:0012186](https://hpo.jax.org/app/browse/term/HP:0012186) is a synonym of Cubital tunnel syndrome - I will add the synonym
